setup.py

```diff
@@ -28,56 +24,47 @@
 # Adapted from Cython's new_build_ext
 class build_ext(_build_ext):
     def finalize_options(self):
+        _build_ext.finalize_options(self)
+
+        # Let the current repository be part of the module search
+        # (otherwise autogen is not found)
+        dir_path = os.path.dirname(os.path.realpath(__file__))
+        sys.path.append(dir_path)
+
         # Generate auto-generated sources from pari.desc
+        from autogen import rebuild
```

Comment:
Does this actually work? In PEP 517 builds, the current directory is not in the search path

Reply:
Oh, I see you're adding it above...

Reply:
nice catch. The CI did not complain. Do you know which pip (or Python) version is concerned?

Reply:
That the current directory is not in the path is by design in PEP 517 builds. Not specific to particular versions.
I think you're working around it correctly. We do the same in sage setup.py